nextflow/workspace_description.md

```diff
@@ -0,0 +1,48 @@
+  > **Note**: If you do not have a GitHub account, then you can clone each repo manually after you've created the Cloud Environment.
+
+
+## Step 1: Preview prior runs of the relevant notebooks.
+
+To gain a better understanding of the setup and analysis included in this demonstration workspace, previews of this notebook that include cell outputs are provided. Navigate to your workspace's Resources tab, then select the "Notebook snapshots" folder. To view a notebook snapshot, select a file in said folder and click the "Preview" button.
+
+## Step 2: Create an Enterprise Terra Cloud Environment and run a setup notebook.
+
+Create an Enterprise Terra Cloud environment, by navigating to the "Environments" tab of the workspace. You can use the configuration defaults.
+
+Launch the environment once it's running, and then run the notebook [workspace_setup.ipynb](https://github.com/DataBiosphere/terra-axon-examples/blob/main/workspace_setup.ipynb). Its repo, `terra-axon-examples`, which is defined as a workspace Git repository, should be automatically cloned to your Enterprise Terra Cloud Environment, and you should be able to navigate to the notebook in the JupyterLab file browser. Look for the `terra-axon-examples` subdirectory.
+
+> **If you do not see the `terra-axon-examples` subdirectory**, open a Terminal window on the notebook server (under the **File** menu) and run:
+>  ```sh
+  git clone https://github.com/DataBiosphere/terra-axon-examples.git
+>  ```
+
+## Step 3: Run the Nextflow examples.
+
+Two examples are provided in `nextflow_examples.ipynb`. You will find this notebook in [the `nextflow` directory of the `terra-axon-examples` repository](https://github.com/https://github.com/DataBiosphere/terra-axon-examples/tree/main/nextflow).  As mentioned above, its repo, `terra-axon-examples`, will be automatically cloned to your Enterprise Terra Cloud Environment, and you should be able to navigate to the notebook in the JupyterLab file browser. Before running either example, you'll need to run the Setup section.
+
+- The first example demonstrates running a Nextflow RNASeq workflow on human gut data.
+- The second example demonstrates running the [`nf-core`](https://nf-co.re/) RNASeq pipeline on yeast genome data.
+
+The examples provided are independent of one another (meaning you can run only the second example if desired, without having to execute the first example).
+
+## Step 4: View MultiQC reports produced.
+
+Running either or both examples will result in the creation of a MultiQC report. To preview what reports can be expected to look like, navigate to the workspace resource folder "Report previews" and click to view:
+1. [Example 1 MultiQC report preview](https://terra-preprod-ui-terra.api.verily.com/workspaces/getting-started-with-workflows-workspace/resources/6cce61b8-4dff-4342-934b-243134d91cac/example_results/multiqc_report.html)
```

Comment:
I think you'll want to do the same thing here with these links that you did with the notebook links

Reply:
Fixed!